src/journal/FutureImpl.h

```diff
@@ -101,7 +100,7 @@ class FutureImpl : public RefCountedObject, boost::noncopyable {
   uint64_t m_entry_tid;
   uint64_t m_commit_tid;
 
-  mutable Mutex m_lock;
+  mutable std::mutex m_lock;
```

Comment:
Nit: why ```std::mutex``` instead of ```ceph::mutex``` as title suggests (and every other commit)?

Reply:
i updated the commit message to explain the reason. i am quoting it here

>  * FutureImpl::m_lock is an exception. as, before this change, the lock
>       was initialized like `m_lock("FutureImpl::m_lock", false, false)`, see
>       the declaration of
>       `Mutex(const std::string &n, bool r = false, bool ld=true, bool bt=false)`
>       so `m_lock` is actually not using the extra features offered by
>       `Mutex` like runtime lockdeps check. and `mutex_debugging_base` does
>       not allow us to disable lockdeps individually. so let's just use the plain
>       `std::mutex`.

Reply:
i switched to a cutomized `ceph::mutex`.